hoomd/hpmc/pytest/test_compute_sdf.py

```diff
@@ -0,0 +1,173 @@
+from __future__ import print_function
+from __future__ import division
+import hoomd
+import pytest
+import numpy as np
+import hoomd.hpmc.pytest.conftest
+
+
+def test_before_attaching():
+    sdf = hoomd.hpmc.compute.SDF(xmax=0.02, dx=1e-4)
+
+    assert sdf.xmax == 0.02
+    assert sdf.dx == 1e-4
```

Comment:
Let's make sure we can set these parameters to some new values and get the new values back.